recipes/LibriSpeech/ASR/CTC/extra_requirements.txt

```diff
@@ -1 +1,4 @@
+k2==1.24.3
```

Comment:
Are those versions compatible with torch>= 2.0?

Reply:
I only used `torch=1.13.1` in my experiments, but from [here](https://k2-fsa.github.io/k2/installation/from_wheels.html#linux-cpu-example) it seems like `k2=1.24.3` is compatible with `torch>=2.0`

Reply:
This [page](https://k2-fsa.github.io/k2/cuda.html) lists all the available versions of k2. I find that the newest version supported by k2 is `2.0.1`. On the other hand, the latest published version on [Pypi](https://pypi.org/project/k2/#history) for Linux is `1.23.4` which relies on `torch=1.13.1`. Thus, when a user would like to install k2 to support `torch>=2.0`, for instance, they have to use `pip install k2==1.24.3.dev20230718+cuda11.7.torch2.0.1 -f https://k2-fsa.github.io/k2/cuda.html` to indicate the source explicitly.